schema-registry/client/src/main/java/com/hortonworks/registries/schemaregistry/client/SchemaRegistryClient.java

```diff
@@ -1397,6 +1404,26 @@ public String run() {
          */
         public static final int DEFAULT_READ_TIMEOUT = 30 * 1000;
 
+        /**
+         * Username for basic authentication.
+         */
+        public static final ConfigEntry<String> BASIC_AUTH_USERNAME =
+                ConfigEntry.optional("schema.registry.client.basic.auth.username",
```

Comment:
minor: I believe clients need not know that basic auth lies underneath. So "schema.registry.auth.username" should be better.

Reply:
fixed